doc/index.rst

```diff
@@ -35,7 +35,7 @@ Usage
 The primary classes are:
 
 * `DomeCsc`: controller for the auxiliary telescope dome.
```

Comment:
Not auxiliary. Consider "dome for the Simonyi Survey Telescope" or something along those lines.

Reply:
Oops, copy and paste. Corrected now!